src/PiggyCustomEnchants/EventListener.php

```diff
@@ -1064,10 +1064,10 @@ public function checkArmorEnchants(Entity $entity, Event $event)
                                 $this->plugin->cloakingcd[strtolower($entity->getName())] = time() + 10;
                                 $effect = Effect::getEffect(Effect::INVISIBILITY);
                                 $effect->setAmplifier(0);
-                                $effect->setDuration(60 * $enchantment->getLevel());
+                                $effect->setDuration(50 * $enchantment->getLevel());
                                 $effect->setVisible(false);
                                 $entity->addEffect($effect);
-                                $entity->sendMessage(TextFormat::DARK_GRAY . "You have become invisible!");
+                                $entity->sendMessage(TextFormat::DARK_GRAY . "You have become invisible!, for 5 seconds!");
```

Comment:
🤦‍♂️ 50 ticks is 2 and a half seconds, not 5

Reply:
Not to mention the duration changes depending on level, while the message doesn't

Reply:
read the top...